src/HaskellCI/GitHub.hs

```diff
@@ -480,6 +488,47 @@ makeGitHub _argv config@Config {..} prj jobs@JobVersions {..} = do
     withHaddock :: String
     withHaddock = "--with-haddock $HADDOCK"
 
+    ircJob :: [String] -> ListBuilder (String, GitHubJob) ()
+    ircJob serverChannelNames = item ("irc", GitHubJob
+        { ghjName            = "Haskell-CI (IRC notification)"
+        , ghjRunsOn          = "ubuntu-18.04"
+        , ghjNeeds           = [mainJobName]
+        , ghjIf              = Nothing
+        , ghjContainer       = Nothing
+        , ghjContinueOnError = Nothing
+        , ghjMatrix          = []
+        , ghjSteps           = concatMap (\serverChannelName -> [ ircStep serverChannelName True
```

Comment:
For the future, I usually use list comprehensions instead of `concatMap`, specially here, where you still need to give element a name,

Reply:
I've rewritten this to use a list comprehension.